example_grpc/protos/katran.proto

```diff
@@ -81,6 +81,18 @@ message hcMap {
     map<int32, string> healthchecks = 1;
 }
 
+message VipReals {
+  Vip vip = 1;
+  string Protocol = 2;
+  string vipFlags = 3;
+  repeated Real reals = 4;
+  
+  }
+  
+  message Services {
+    map<string, VipReals> vipReals = 1;  
+  }
+
```

Comment:
This is not needed.

Reply:
I need it to prevent omitempty. because when I use normal json.Marshal the default fields are not present in json dump because of (json:"address,omitempty").
Do you have a better solution?